src/store/modules/product/actions.ts

```diff
@@ -40,12 +40,48 @@ const actions: ActionTree<ProductState, RootState> = {
       // Remove added loader only when new query and not the infinite scroll
       if (payload.viewIndex === 0) emitter.emit("dismissLoader");
     } catch(error){
-      console.log(error)
```

Comment:
Why we have removed this console statement?

Reply:
Okay sir I have added this console.log()

Reply:
But this console statement is not added by you.